lib/id3c/cli/command/de-identify.py

```diff
@@ -0,0 +1,89 @@
+"""
+De-identify personally identifiable information (PII) by generating a hash.
+"""
+import click
+import logging
+import os
+import hashlib
+import pandas as pd
+from sys import stdout
+from typing import Tuple
+from id3c.cli import cli
+from id3c.cli.command import load_input_from_file_or_stdin
+
+
+LOG = logging.getLogger(__name__)
+SUPPORTED_EXTENSIONS = ('.csv', '.tsv', '.xls', '.xlsx')
+
+@cli.command("de-identify")
+@click.argument("columns",
+    metavar = "<columns>",
+    nargs = -1)
+@click.option("--filename",
+    metavar = "<filename.{csv,tsv,xls,xlsx}>",
+    type = click.Path(exists=True))
```

Comment:
I think a more conventional/Unixy interface here would be `id3c de-identify column1 column2 path/to/file.csv` or `id3c de-identify column1 column2 -` for stdin. This can be supported with something like:

```python
@click.option("filename",
    metavar = "<filename.{csv,tsv,xls,xlsx}>",
    type = click.File("r"))
```